[IMP] Events: Add event form PoS sales smart button (#18055)
odoo/documentation: content/applications/marketing/events/promote_monetize/sell_tickets.rst

Reviewer: ```suggestion
be set to :guilabel:`Service`, and the :guilabel:`Create on Order` field **must** be set to
```
suggestion: added comma

Reviewer: ```suggestion
With the products configured, navigate the to :menuselection:`Point of Sale` app and open a :abbr:`POS
```
suggestion: added the

.. image:: sell_tickets/events-settings-tickets.png
   :alt: View of the settings page for Odoo Events.

With these settings enabled, any event with paid tickets sold either through a sales order or
through the website features a :icon:`fa-dollar` :guilabel:`Sales` smart button at the top of the

> event form. Additionally, a :icon:`fa-shopping-basket` :guilabel:`PoS Sales` smart button also

Reviewer: ```suggestion
event form. Additionally, a :icon:`fa-shopping-basket` :guilabel:`PoS Sales` smart button
```
suggestion: removed also.

appears if any tickets are sold in a :abbr:`POS (point of sale)`.

Clicking the :icon:`fa-dollar` :guilabel:`Sales` or the :icon:`fa-shopping-basket` :guilabel:`PoS
Sales` smart button reveals a separate page, showcasing all the sales orders or :abbr:`POS (point of

> sale)` orders related to tickets that have been sold for that specific event.

Reviewer: ```suggestion
Clicking the :icon:`fa-dollar` :guilabel:`Sales` or :icon:`fa-shopping-basket` :guilabel:`PoS
Sales` smart button opens a separate page that shows all the sales or :abbr:`POS (point of
sale)` orders related to tickets sold for that specific event.
```
Revised this to be more concise.
